src/components/pages/test-pages/comment-page/CommentTestPage.styles.ts

```diff
@@ -0,0 +1,12 @@
+import { SxProps, Theme } from '@mui/material/styles';
+
+const wrapper: SxProps<Theme> = {
+  width: '100%',
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  gap: '10px',
+};
+
+export { wrapper };
```

Comment:
Actually, it is not necessary to write export like it. You can just write `export`  before `const`, result is the same. Notice, that If you would have huge amount of styles, you will write looooong export.
P.S : same issue in Comment.styles